tools/memory_estimator/json_report_generator.py

```diff
@@ -0,0 +1,82 @@
+import os
+import json
+from operator import truediv
+
+
+# Some of the file names are modified when diplaying on the website. This dict
```

Comment:
minor: I think we can make this string better. # Mapping of original file name to customized name with additional information.

Reply:
Done.